.github/workflows/ci.yml

```diff
@@ -43,6 +43,7 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install gcc-arm-linux-gnueabihf
+          rustup target add armv7-unknown-linux-gnueabihf
```

Comment:
If you add a rust-toolchain.toml to the repo like we do in xmr-btc, then you don't even have to add this line! :)

Reply:
https://github.com/comit-network/rendezvous-server/runs/2969565537
doesn't work

Reply:
I don't see a `rust-toolchain.toml` file [in the repository at the commit](https://github.com/comit-network/rendezvous-server/tree/7c5250d4589cca73a293b9fa11e8c8fe7a9b6fd4) that the workflow you linked to ran at.

Reply:
I dropped the commit and force pushed. I will try again and leave the commit up to make sure